### PR TITLE
Add more control over senders

### PR DESCRIPTION
**Pull request by OrlandoCo**
#### Description

This pull requests it's to add more control over subs:
1. Add nack rate limit from subs to pubs
2. Enable/disable remb from subs to pubs
3. Subs should respect max bandwidth from SFU

### CODE REVIEW

File: pkg/router.go [hunk 1]
```diff
@@ -4,19 +4,26 @@ import (
...
+	"time"
 
 	"github.com/pion/ion-sfu/pkg/log"
 	"github.com/pion/rtcp"
 	"github.com/pion/webrtc/v3"
 )
 
+const ()
+
+var routerConfig RouterConfig
```
Comment (OrlandoCo) [inline pkg/router.go:17]: A global router config was added to not pass the same configuration through many functions.

File: config.toml [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
-minbandwidth = 100000
-maxbandwidth = 5000000
+# pass bandwidth feeback from subs to pubs
+subrembfeedback = false
+# Limit the remb bandwidth in kbps
+# zero means no limits
+maxbandwidth = 1000
+# Rate limit nack packets from senders to 1 nack per maxNackTime in seconds
+# zero means no rate limit
+maxNackTime = 1
```
Comment (tarrencev) [inline config.toml]: `maxnacktime` for consistency

File: pkg/sender.go [hunk 3]
```diff
@@ -41,21 +41,23 @@ type WebRTCSender struct {
...
+		sendChan:   make(chan *rtp.Packet, maxSize),
 	}
 
 	for _, feedback := range track.Codec().RTCPFeedback {
 		switch feedback.Type {
 		case webrtc.TypeRTCPFBGoogREMB:
 			log.Debugf("Using sender feedback %s", webrtc.TypeRTCPFBGoogREMB)
-			s.useRemb = true
-			go s.rembLoop()
+			if routerConfig.REMBFeedback {
```
Comment (tarrencev) [inline pkg/sender.go:57]: log above should go into the `if`

**Review by tarrencev — APPROVED** [review]
minor comments otherwise lgtm!

File: config.toml (hunk 2 above)
Comment (OrlandoCo) [reply config.toml]: done.

**Pull request merged**